lib/doc/worksheet.js

```diff
@@ -563,6 +563,23 @@ class Worksheet {
   }
 
   // =========================================================================
+
+  // Style fill
+  fillCells(range, fill) {
+    const dimensions = new Range(range); // convert arguments into Array
+    // apply style
+    for (let i = dimensions.top; i <= dimensions.bottom; i++) {
+      for (let j = dimensions.left; j <= dimensions.right; j++) {
+        if (i >= dimensions.top || j > dimensions.left) {
+          const cell =this.getCell(i, j);
+          cell.fill = fill;
```

Comment:
should be deep copy here.
In other case, changes in one of those cells should propagate for all others from `range`.

Reply:
If I could please You, make a test for that scenario :)

Reply:
Should I use cloneDeep from spec\utils\under-dash.js ?

Reply:
`spec` folder is for tests, so if you want to use this function it's should be moved to `lib/utils/under-dash.js`